ICSharpCode.Decompiler.PowerShell/GetDecompilerCmdlet.cs

```diff
@@ -15,20 +25,106 @@ public class GetDecompilerCmdlet : PSCmdlet
 		[ValidateNotNullOrEmpty]
 		public string LiteralPath { get; set; }
 
+		[Parameter(Mandatory = false, HelpMessage = "file with decompiler settings")]
+		public string SettingsPath { get; set; }
+
 		protected override void ProcessRecord()
 		{
 			string path = GetUnresolvedProviderPathFromPSPath(LiteralPath);
 
-			try {
-				var decompiler = new CSharpDecompiler(path, new DecompilerSettings() {
-					ThrowOnAssemblyResolveErrors = false
-				});
+			try
```

Comment:
This does not respect our code formatting style. Please make sure you follow the settings in .editorconfig.